packages/orchestration/src/orchestration-stream-chunk-response.ts

```diff
@@ -26,9 +27,12 @@ export class OrchestrationStreamChunkResponse {
    * @returns The finish reason.
    */
   getFinishReason(choiceIndex = 0): string | undefined {
-    return this.getChoices()?.find(
-      (c: LlmChoiceStreaming) => c.index === choiceIndex
-    )?.finish_reason;
+    return this.getChoices()?.find(c => c.index === choiceIndex)?.finish_reason;
+  }
+
+  getToolCalls(choiceIndex = 0): ToolCallChunk[] | undefined {
```

Comment:
```suggestion
  getDeltaToolCalls(choiceIndex = 0): ToolCallChunk[] | undefined {
```
It was a bit confusing as ToolCalls sound complete already but it's returning chunks.